Modules/Module_Processing/Module_Population/xASL_im_CreateAnalysisMask.m

```diff
@@ -49,10 +49,11 @@
 bSkipStandard = 0;
 
 if x.dataset.nSubjectsSessions<16
-    fprintf('%s\n', ['Too few subjects (' num2str(x.dataset.nSubjectsSessions) ') to create population-based analysis mask']);
-    x.S.MaskSusceptibility = xASL_im_IM2Column(ones(121,145,121),x.S.masks.WBmask);
-    x.S.VBAmask = x.S.MaskSusceptibility;
-    bSkipStandard = 1;
+    % With too small datasets, created templated are not reliable
+    fprintf('\n\n%s\n\n', ['Only n=' num2str(x.dataset.nSubjectsSessions) ' subject*runs, population-based analysis mask may not be useful']);
+    % x.S.MaskSusceptibility = xASL_im_IM2Column(ones(121,145,121),x.S.masks.WBmask);
+    % x.S.VBAmask = x.S.MaskSusceptibility;
+    % bSkipStandard = 1;
 elseif isempty(PathSusceptibilityMask)

```

Comment:
WHy are we checking for PathSusceptibilityMask - we are creating that here, so it should be always empty...

Reply:
No, here we load the template, and we create the group mask later. Shall I also change this variable name in this function to clarify this?

E.g.:

`PathSusceptibilityMask` -> `pathTemplateSusceptibilityMask`